File: Python/programmers/etc/level1/푸드_파이트_대회.py
def solution(food):
    answer = ""
    for idx, f in enumerate(food):
        answer += str(idx)*(f//2)
    answer += "0" + answer[::-1]
    return answer


food = [1, 3, 4, 6]
print(solution(food)) # "1223330333221"

food = [1, 7, 1, 2]
print(solution(food)) # "111303111"

File: Java/programmers/etc/level1/푸드파이트대회.java
package CodeTest.Java.programmers.etc.level1;

public class 푸드파이트대회 {
    public String solution(int[] food) {
        String answer = "0";
        
        for (int i=food.length-1; i>=0; i--) {
            for (int j=0; j<food[i]/2; j++) {
                answer = i+answer+i;
            }
        }
        return answer;
    }
}
